jackson-simple/src/test/java/com/baeldung/jackson/date/Event.java

```diff
@@ -4,17 +4,17 @@
 
 import com.fasterxml.jackson.annotation.JsonFormat;
 
-public class EventWithFormat {
+public class Event {
```

Comment:
I think the EventWithFormat is for a different article; instead we should have the EventWithSerializer in the article

Reply:
Done, reverted the rename